templates/traefik.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       kubeaddons.mesosphere.io/provides: defaultstorageclass
   chartReference:
     chart: traefik
-    repo: https://mesosphere.github.io/charts/staging
+    repo: https://timaa2k.github.io/charts
```

Comment:
let's revert https://mesosphere.github.io/charts/staging and bump to the right chart version